### ISSUE TITLE
permissions.json isn't pulled into the docker container

### ISSUE THREAD

**Issue by teflontoni** [IMetZach/bedrock-server#1]
Hey, I've forked your excellent little repository and bumped it to the newest available bedrock server version.

As someone who hasn't done nothing with docker, I'm proud it's working somehow for me...

```
#!/bin/bash

server=$1
base_dir="/var/games"
work_dir="${base_dir}"/"${server}"

mkdir -p "${work_dir}"
touch "${work_dir}"/whitelist.json
touch "${work_dir}"/permissions.json
touch "${work_dir}"/server.properties

docker run -d -h ${server} --name=${server}\
 -v "${work_dir}/whitelist.json:/opt/minecraft/whitelist.json:z"\
 -v "${work_dir}/permissions.json:/opt/minecraft/permissions.json:z"\
 -v "${work_dir}/server.properties:/opt/minecraft/server.properties:z"\ 
 -v "${work_dir}/worlds:/opt/minecraft/worlds:z"\
  --network=host\
 teflontoni/minecraft-bedrockserver
```

I deleted the ops.json reference, because the bds can't interprete it/ignores it.

So the way to go is permissions.json. But my permissions.json is empty in the container. whitelist and server.properties are there in my customized layout, even my world from my second server. But this permissions thing is driving me (and my kids) insane.

What do I wrong? Any hints are deeply appreciated.

**Comment by IMetZach** [comment]
Could you share what your changes look like and the error you're receiving, please?

**Comment by teflontoni** [comment]
Just skimmed my log, docker doesn't complain about anything. Apart from minecraft_launcher.sh pasted above I've edited Dockerfile as follows:

```
FROM fedora:29

LABEL version="1.11.2.1"

ENV NAME Bedrock-Server
ENV arc=bedrock-server-1.11.2.1.zip
ENV dlarc=https://minecraft.azureedge.net/bin-linux/${arc}

WORKDIR /opt/minecraft

RUN ["dnf", "-y", "upgrade"]
RUN ["dnf", "-y", "install", "unzip" ]
RUN ["dnf", "clean", "all"]
RUN ["rm", "-rf", "/var/cache/dnf"]
RUN ["mkdir", "/opt/minecraft/worlds"]

# VOLUME ["/opt/minecraft/permissions.json"]
# VOLUME ["/opt/minecraft/server.properties"]
# VOLUME ["/opt/minecraft/whitelist.json"]
# VOLUME ["/opt/minecraft/worlds"]

ADD ${dlarc} ${arc}
RUN unzip -n ${arc}

ADD ./startup.sh /opt/minecraft/
RUN ["chmod", "+x", "/opt/minecraft/startup.sh"]

EXPOSE 19132
EXPOSE 19133

ENTRYPOINT ["/opt/minecraft/startup.sh", "/bin/bash"]
```
The changes just reflect the version bump...

I tried touching permissions.json again after stoping the container and than restarted the docker container, but permissions.json inside the container stays empty. 

I edited the file via vi directly in the container and did a restart, to no avail, too.

If you need any further debug output, I'll do that.


EDIT: In case it's unclear, people who should be OP on the server aren't because the permissions.json file inside the docker container is empty (just two lonely [ ]).

Expected file content somthing like

```
[
{
        "permission": "operator",
        "xuid": "1234567890123456"
}
]
```

**Comment by teflontoni** [comment]
Just deleted permissions.json in /var/games/XXXX
and after `docker stop XXXX ` even the empty permissions.json doesn't show up in /var/games/XXXX

So docker seems to ignore the -v mount completely for that file... can you reproduce that behaviour?

**Comment by IMetZach** [comment]
Let me see what I can do. I was running Fedora Atomic Host on a server at home and it was killed by a power outage. I'll get a VM going and see what happens. What version of Linux are you running it on? Also, the Docker container won't create those files outside the container, that's why the launcher script touches the files.

**Comment by teflontoni** [comment]
I copied all files from my other server for testing purposes, whitelist.json, server.properties and world all worked out of the box.

I run the latest docker version on a fully patched CentOS 7 Server. 

What kinda worked: I edited permissions.json directly in the docker image (container stopped, of course). Restarted the container and got OPed...

But that's an very ugly solution and will break for sure with each dot point release...

**Comment by teflontoni** [comment]
😢
okay, it was a long day yesterday and I'm pretty sure I derped hard...

It's no secret that changing something on a script in a repository makes it a necessity to clone these changes to your actual server...

I'm sorry if I ruined your weekend and bow my head in shame, embarassed. 

Between all the testing and optimizing I've completely lost track of minecraft-server.sh
So in github everything was dandy, on my server the poor .sh file was the good ol' self.

I'm really sorry.
